app/views/users/show.html.erb

```diff
@@ -4,3 +4,45 @@
   <%= link_to "Log a Meal", new_meal_path, class: 'btn btn-info' %>
   <%= link_to "See My Results", results_path, class: 'btn btn-success' %>
 </p>
+
+<h3 class='m-4 text-center'>Your Meals</h3>
+<section class="meals px-5">
+  <% if @meals.any? %>
+    <table class="table table-striped table-bordered" id="user-meals">
+      <thead>
+        <tr>
+          <th scope="col">Meal Title</th>
+          <th scope="col">Date Eaten</th>
+          <th scope="col" class='text-center'>Gut Feeling</th>
+
+          <th scope="col" class='text-center'>Delete</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @meals.each do |meal| %>
+          <tr id="meal-<%= meal.id %>">
+            <td><%= meal.title %></td>
+            <td><%= localtime(meal) %></td>
+            <% if meal.gut_feeling && !one_day_old?(meal)%>
+              <td class='text-center'><strong><%= meal.gut_feeling %></strong><br>
+                <%= link_to 'Update', edit_meal_path(meal), class: 'btn btn-sm btn-warning' %></td>
+            <% elsif !meal.gut_feeling && !three_days_old?(meal) %>
+              <td class='text-center'><%= link_to 'Add Gut Feeling', edit_meal_path(meal), class: 'btn btn-sm btn-info' %></td>
+            <% else %>
+              <td class='text-center'><strong><%= meal.gut_feeling %></strong></td>
+            <% end %>
+            <td class='text-center'><%= button_to 'Delete', meal_path(meal), method: :delete, class: 'btn btn-sm btn-danger' %></td>
```

Comment:
Should the user be able to delete a meal? We need as much information as possible to create accurate results for the users. Being able to edit a meal is important since the user may have entered something wrong, but being able to fully delete a meal could have negative consequences on the accuracy of our program.